web/src/components/Button/SmallCircleButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+
+import styled, { DefaultTheme } from 'styled-components';
+
+const Button = styled.button`
+  font-weight: 900;
+  text-align: center;
+  padding: 10px;
+  color: ${(props) => props.theme.colors.text};
+  background-color: ${(props) => props.theme.colors.button};
+  width: 35px;
+  height: 35px;
+  border-radius: 50%;
+  border: none;
+  transition: 250ms background ease;
+  cursor: pointer;
+
+  &:hover {
+    opacity: 0.8;
+  }
+`;
+
+interface SmallCircleButtonProps {
```

Comment:
Same above. `Circle` makes sense but **how small** it is? If this is universal component, then something general would fit